roles/ovirt-host-deploy-libvirt-guests/tasks/main.yml

```diff
@@ -0,0 +1,35 @@
+---
+- name: Fetch info about VDSM
+  command: bash -c "rpm -qi vdsm | grep -oE 'Version\\s+:\\s+[0-9\\.]+' | awk '{print $3}'"
+  changed_when: false
+  register: vdsm_version
+
+- block:
+    - name: Change default timeout if setup
+      ini_file:
+        state: present
+        section: null
+        path: /etc/sysconfig/libvirt-guests
+        option: "SHUTDOWN_TIMEOUT"
+        value: "{{ host_deploy_shutdown_timeout }}"
+      when:
+        - host_deploy_shutdown_timeout is defined
```

Comment:
So whatabout the defaults, you,ve mentioned in readme?

Reply:
With current deployment the defaults handling is tricky. The default values are given by controller from VDSM itself during VDSM deploy phase on the host.
This opens up a discussion for where to start putting the configuration, should it stay in the otopi plugin inside the VDSM or should it be completely handled by ansible?
@michalskrivanek  @mwperina what do you think?

Reply:
@petrkotas where is it in otopi plugins?

Reply:
Ok, then I think you should remove the defaults from the README, and say it don't have any default value. and also please remove the `default` directory.

Reply:
@michalskrivanek it is placed in configurators https://github.com/oVirt/vdsm/blob/master/lib/vdsm/tool/configurators/libvirt.py this is the libvirt one. The libvirt guests configuration can be seen in my pull request on gerrit https://gerrit.ovirt.org/#/c/79840/15/lib/vdsm/tool/configurators/libvirt.py

My opinion is, that once we fully transition towards the ansible. We can completely omit the old controllers and use ansible to fully control the host.

What do you think?

Reply:
then why bother with 79840?